.github/pull_request_template.md

```diff
@@ -20,7 +20,6 @@
 
 ## :pencil: Checklist
 <!--- Put an `x` in the boxes that apply -->
-- [ ] I ran `./gradlew spotlessApply` before submitting the PR
 - [ ] I reviewed submitted code
```

Comment:
Should we add a task that asks contributors to reformat their code using the provided IDE style config?
Also we need to ensure that the license identifiers get added to new files.

Reply:
Ack on both fronts. On it.

Reply:
Rebased and force pushed. Added copyright config to the first commit, updated PR template to add a formatting task with the second commit.